src/lib/vuetilities.js

```diff
@@ -89,6 +91,18 @@ export function vModelFromNode(el) {
   return normalizeKeyPath(vModelName);
 }
 
+const renderNode = {
+  props: ['node'],
+  /**
+   * @returns {*} The node being forced to render
+   */
+  render() {
+    return this.node ? this.node : null;
```

Comment:
Is this a better way to do this?
```suggestion
    return this.node || null;
```